app/mailers/devise_mailer.rb

```diff
@@ -42,6 +42,8 @@ def invitation_instructions(record, token, opts={})
       elsif record.team_members.last.role == 'basic'
         set_template(Rails.application.secrets.notify_basic_invite_template)
         set_standard_personalisations(record, token)
+      else
+        raise PermissionTitleError, 'You must select a valid \'Permission Title\''
```

Comment:
If you double quote the string you don't need to escape e.g. `"You must select a 'Permission Title'"`